packages/flutter/lib/src/widgets/scroll_position_with_single_context.dart

```diff
@@ -238,6 +245,7 @@ class ScrollPositionWithSingleContext extends ScrollPosition implements ScrollAc
       delegate: this,
       details: details,
       onDragCanceled: onDragCanceled,
+      carriedVelocity: context.momentumCarryFunction(_heldPreviousVelocity),
```

Comment:
why do you get this from the context rather than just calling it on the physics directly?